documentation/src/main/asciidoc/introduction/Configuration.adoc

```diff
@@ -175,7 +176,7 @@ Each `<class>` element specifies the fully-qualified name of an entity class.
 
 .Scanning for entity classes
 ****
-In some container environments, for example, in any EE container, the `<class>` elements are unnecessary, since the container will scan the archive for annotated classes, and automatically recognize any class annotated `@Entity`.
+In some container environments, for example, in any EE container, the `<class>` elements are unnecessary, since the container will scan the archive for annotated classes, and automatically recognize any class annotated with `@Entity`.
```

Comment:
I prefer just "annotated `@Entity`". 

"With" is unnecessary here and sounds unnatural to me. I would never use "with" here in regular speech.